docs/modules/ROOT/pages/how-tos/testing/integration/adding.adoc

```diff
@@ -17,6 +17,7 @@ Complete the following steps in the {ProductName} console:
 . In the *Integration test name* field, enter a name of your choosing.
 
 . In the *GitHub URL* field, enter the URL of the GitHub repository that contains the test you want to use.
+NOTE: If you do not already have a custom test that you want to add, we tell you how to xref:./creating.adoc[create a custom integration test] in the next section.
 . Optional: If you want to use a branch, commit, or version other than the default, specify the branch name, commit SHA, or tag in the *Revisions* field.
```

Comment:
The note is not showing up properly on the render: https://pr-88--konflux-docs.netlify.app/how-tos/testing/integration/adding/ Maybe we need to add some whitespace
```suggestion
. In the *GitHub URL* field, enter the URL of the GitHub repository that contains the test you want to use.

+
NOTE: If you do not already have a custom test that you want to add, we tell you how to xref:./creating.adoc[create a custom integration test] in the next section.

. Optional: If you want to use a branch, commit, or version other than the default, specify the branch name, commit SHA, or tag in the *Revisions* field.
```
Ah, and the note below has the `+` on the preceding line.